README.md

```diff
@@ -1,12 +1,18 @@
 # Numenta research papers code and data
 This repository contains reproducible code for selected Numenta papers. It is currently under construction and will eventually include the source code for all the scripts used in Numenta's papers.
 
+### Avoiding Catastrophe: Active Dendrites Enable Multi-task Learning in Dynamic Environments
+In this paper we show that the biophysical properties of dendrites, synapses, and local inhibitory systems enable networks to dynamically restrict and route information in a context-specific manner. First, we propose a novel artificial neural network architecture that incorporates active dendrites and sparse representations into the standard deep learning framework. Next, we study the performance of this architecture in two separate benchmarks requiring task-based adaptation: Meta-World, a multi-task reinforcement learning environment where a robotic agent must learn to solve a variety of manipulation tasks simultaneously; and a continual learning benchmark in which the model’s prediction task changes throughout training.
+> [Source: multitask reinforcement learning][10_src]
+
+> [Source: continual supervised learning][8_src]
+
```

Comment:
I think these lines should be replaced with:
```
[Sources][10_src]
```

Reply:
There are two repositories linked to the paper. Refer to discussion in slack

Reply:
This is confusing. The reader won't have access to the slack discussion for context. I am not sure I know what slack discussion you are referring to. 

The reader should not be confused reading this file and not knowing what to click next. It should be only one source code link per paper on this page.

If you need to add a reference to another paper you can add that reference in the paper's [README](https://github.com/numenta/nupic.embodied/blob/master/projects/multitask/README.md), not in the main [README](https://github.com/lucasosouza/htmpapers/blob/mtrl/README.md. 

See how it was done in [Grid Cell Path Integration For Movement-Based Visual Object Recognition](https://github.com/numenta/htmpapers/tree/master/bmvc/grid_cell_path_integration_for_movement_based_visual_object_recognition)

Reply:
It is different here. It is two projects, one paper. There is no one primary project.

Reply:
In that case you should merge the sources or create 2 entries (part 1, part 2). It is confusing to see 2 source links with one paper. I would not know what to click next

Reply:
we could do two entries

Reply:
Closing as this is no longer relevant, has been fixed